File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/kafka/producers/model/SvarAnmodningUnntak.java
package no.nav.melosys.eessi.kafka.producers.model;

import lombok.Data;
import no.nav.melosys.eessi.models.sed.medlemskap.impl.SvarAnmodningUnntakBeslutning;

@Data
public class SvarAnmodningUnntak {

    private SvarAnmodningUnntakBeslutning beslutning;
    private String begrunnelse;
    private Periode delvisInnvilgetPeriode;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/kafka/producers/model/SvarAnmodningUnntak.kt
package no.nav.melosys.eessi.kafka.producers.model

import no.nav.melosys.eessi.models.sed.medlemskap.impl.SvarAnmodningUnntakBeslutning

data class SvarAnmodningUnntak(
    var beslutning: SvarAnmodningUnntakBeslutning? = null,
    var begrunnelse: String? = null,
    var delvisInnvilgetPeriode: Periode? = null
)
